<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeomConstraints</class>
 <widget class="QWidget" name="GeomConstraints">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Geometrical constraints</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QTableWidget" name="tw_constraints_list">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>New Row</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Source component</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Target component</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="verticalGroupBox">
     <property name="title">
      <string>Properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gl_properties">
        <item row="3" column="2">
         <widget class="QComboBox" name="cb_motion_direction"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="l_label_motion_direction">
          <property name="text">
           <string>Motion direction:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="l_label_constraints">
          <property name="text">
           <string>No. constraints:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="l_constraints">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="l_label_animations">
       <property name="text">
        <string>Enable animations:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_animations">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
